test/integration/event-handler-test.js

```diff
@@ -1,13 +1,13 @@
 const test = require("tap").test;
 
-const EventHandler = require("../../src/event-handler");
+const createEventHandler = require("../../pkg/dist-src/event-handler");
```

Comment:
don't use the `dist-src` folder in `pkg`, instead use the export from `require("../../pkg")`.

Reply:
I think I'm not following you here. When I run `pika build` I get this folder structure generated so it's not possible to apply the path you suggest:

![image](https://user-images.githubusercontent.com/2574275/79623310-dbcd8200-811b-11ea-96f2-bc253bd0d4b5.png)

In any case, since it's a temporary path until the TypeScript migration is finished, is it important to not use `dist-src`? 

Thanks @gr2m !

Reply:
`pkg/package.json` points to `dist-node/index.js`

Reply:
> `pkg/package.json` points to `dist-node/index.js`

Sorry but even it's there I'm not able to make it work. I added all the code https://github.com/octokit/webhooks.js/pull/113 alrerady. Since it's not merged yet, maybe pika's package is still not available? Not sure what I'm missing.

@gr2m